app/assets/javascripts/components/student/student_page.es6.jsx

```diff
@@ -42,7 +42,7 @@ class StudentPage extends Component {
         <div style={this.styles.container}>
           <Sidebar shouldShow={this.state.sidebar} />
           <StudentGrid {...this.props} />
-          <div style={this.styles.placeholder}></div>
+          <StudentComments comments={[{user: "Max Wolffe", content: "Cats and Dogs and Mice"},]}/>
```

Comment:
Missing a space before the `/>`! Maybe let's give this a new line for the props?